### ISSUE TITLE
Error opening [FTL]

### ISSUE THREAD

**Issue by KryptoPhysX** [Wunkolo/UWPDumper#32]
How i can get it to work ?
![Capture d’écran (165)](https://user-images.githubusercontent.com/63559719/79077998-99640980-7d05-11ea-835b-b94a5a522c01.png)

**Comment by KryptoPhysX** [comment]
i am on windows 10 pro v.1903  ,  os 18362.720

**Comment by Wunkolo** [comment]
It looks like it successfully dumped based on the log there.

It says clearly it should be at
`C:\Users\Arthur\AppData\Local\Packages\SubsetGames.FTLFasterThanLight_gvagsjwfgyhyc\TempState\DUMP`

**Comment by SuperMario20** [comment]
I don't think it finished dumping. Due to the errors reading certain files, the dump will likely be incomplete.

**Comment by KryptoPhysX** [comment]
the DUMP file is empty :/

**Comment by Wunkolo** [comment]
Ah I see now. It's partially reading the files for some reason. This is similarly happening on MCC with #31. I haven't been able to diagnose much of this UWP stuff due to my main workstation suffering an AIO leak but part of me thinks that there is some access control stuff in place to stop certain kinds of dumping as only _some_ files refuse to be dumped despite it reading the file into memory and then writing it, as opposed to a filesystem operation.

https://github.com/Wunkolo/UWPDumper/blob/9fb0a040e674521c1413276bcea6e4e708f34d19/UWPDumper/main.cpp#L156

**Comment by Cizlin** [comment]
I am encountering the same issue with MCC, and after looking into it, I agree that it is probably due to access control. The dumper can successfully dump all files that I can access with a simple owner change. However, if I cannot change the owner to access a file, then the dumper cannot access the file either.

**Comment by SuperMario20** [comment]
This might help with taking ownershop of directories: https://www.howtogeek.com/howto/windows-vista/add-take-ownership-to-explorer-right-click-menu-in-vista/
